### PR TITLE
M2-1744 Change create applet api release

### PR DESCRIPTION
**Pull request by wildkiomi**

### CODE REVIEW

**Review by muhurov — APPROVED** [review]

File: src/shared/state/Applet/Applet.thunk.ts [hunk 1]
```diff
@@ -40,10 +40,10 @@ export const getAppletWithItems = createAsyncThunk(
 );
 
 export const createApplet = createAsyncThunk(
-  AppletThunkTypePrefix.Create,
-  async (body: SingleApplet, { rejectWithValue, signal }) => {
+  AppletThunkTypePrefix.Update,
```
Comment (alex-i-e) [inline src/shared/state/Applet/Applet.thunk.ts:43]: why did you change this:
```suggestion
  AppletThunkTypePrefix.Create,
```

**Review by IhorNesterenko99 — APPROVED** [review]
